test/functional/dbcrash.py

```diff
@@ -175,9 +168,8 @@ def verify_utxo_hash(self):
                 nodei_utxo_hash = self.restart_node(i, self.nodes[3].getbestblockhash())
             assert_equal(nodei_utxo_hash, node3_utxo_hash)
 
-
     def generate_small_transactions(self, node, count, utxo_list):
-        FEE = 1000 # TODO: replace this with node relay fee based calculation
+        FEE = 1000  # TODO: replace this with node relay fee based calculation
```

Comment:
Feel free to address the TODO as well :)  I think you just do something like `relayfee = self.nodes[0].getnetworkinfo()['relayfee']`, and then convert to the right units.